core/src/main/java/org/apache/calcite/rel/rules/PruneEmptyRules.java

```diff
@@ -540,4 +540,32 @@ public interface JoinRightEmptyRuleConfig extends PruneEmptyRule.Config {
       };
     }
   }
+
+
+  public static final RelOptRule EMPTY_TABLE =
```

Comment:
All other rules in this class are suffixed with `INSTANCE`. To keep things uniform I would suggest one of the following:
* `EMPTY_TABLE_INSTANCE`
* `TABLE_ZERO_ROWS_INSTANCE`
* `ZERO_ROWS_TABLE_INSTANCE`